.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -110,6 +110,25 @@ jobs:
     buildSteps:
       - template: template-skipvstest.yml
       - bash: |
+          (
+            while true
+            do
+              sleep 120
+              now=$(date +%s)
+              pids=$(ps aux | grep -v grep | grep -E "^.{,100}docker build" | awk '{print$2}')
```

Comment:
You want to list the docker PIDs, right? A small improvement.
ps -C docker -o pid,args | grep "docker build"

Reply:
I think 'ps -C' is good. But etime is not good to use.
etime has a format '[[dd-]hh:]mm:ss'. If I want to compare it with 1 hour, I need to compare the etime length.